paasta_tools/cli/cmds/spark_run.py

```diff
@@ -36,6 +36,30 @@
 DEFAULT_SPARK_MESOS_SECRET_FILE = '/nail/etc/paasta_spark_secret'
 
 
+deprecated_opts = {
+    '--jars': 'spark.jars',
```

Comment:
Pretty sure you don't want the `--` prefix on this...and might also want `j` here?